core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java

```diff
@@ -1,5 +1,7 @@
 package com.scalar.db.storage.cosmos;
```

Comment:
Do we need to quote the following:
https://github.com/scalar-labs/scalardb/blob/fix/cassandra-admin-reserved-keywords/core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java#L137
https://github.com/scalar-labs/scalardb/blob/fix/cassandra-admin-reserved-keywords/core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java#L166
https://github.com/scalar-labs/scalardb/blob/fix/cassandra-admin-reserved-keywords/core/src/main/java/com/scalar/db/storage/cosmos/SelectStatementHandler.java#L200

If yes, we need to add tests for them to detect these misses.

Reply:
Thank for pointing it out. We need to quote them. I fixed it and added tests for them as well.